packages/babel-traverse/src/index.js

```diff
@@ -33,6 +33,7 @@ traverse.explode = visitors.explode;
 traverse.NodePath = require("./path");
 traverse.Scope    = require("./scope");
 traverse.Hub      = require("./hub");
+traverse.cache    = require("./cache");
```

Comment:
yeah, I agree

On Sun, Mar 6, 2016 at 8:22 PM Logan Smyth <notifications@github.com> wrote:

> In packages/babel-traverse/src/index.js
> <https://github.com/babel/babel/pull/3393#discussion_r55161119>:
>
> > @@ -33,6 +33,7 @@ traverse.explode = visitors.explode;
> >  traverse.NodePath = require("./path");
> >  traverse.Scope    = require("./scope");
> >  traverse.Hub      = require("./hub");
> > +traverse.cache    = require("./cache");
>
> I think the core-js polyfill will serve us fine in this case since it does
> essentially what we were already doing, but I would love to avoid exposing
> this publicly since don't *think* we need to?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/babel/babel/pull/3393/files#r55161119>.
>